t/create-withdrawal-data.uncompressed.decoded

```diff
@@ -1,6 +1,6 @@
 {
-  "txid": "cc0871827f83c927d79ca8d50c52a72fcaf4223f1de7f92931bfd70f7d44e3b3",
-  "hash": "cc0871827f83c927d79ca8d50c52a72fcaf4223f1de7f92931bfd70f7d44e3b3",
+  "txid": "31906179a4f3d983f62cecd9e8fcd4bc3df4969e3d5ec85547b3fecbdb53c51c",
+  "hash": "31906179a4f3d983f62cecd9e8fcd4bc3df4969e3d5ec85547b3fecbdb53c51c",
```

Comment:
why did all of these fixtures change?  Should they not be compatible with the script after the version bump?

Reply:
This is explained in the commit message for 60e189b4883cbe6ea43e2304a02157372d53c764. The order of the transaction outputs was dependent on the ordering of a python dict, which is not specified by the language, and effectively random. It may change when the python interpreter version changes. To alleviate this, I switched to using an OrderedDict, and now the ordering of the outputs is consistent.

Reply:
got it.  How did we not have tests fail from the unordered python dict before?  Please let me know if I'm missing something

Reply:
[More than you ever wanted to know about Python dict ordering](https://stackoverflow.com/a/15479974)

In 2.7 it was stable but unspecified. Every run would be the same. The Python3 version we have (3.5.2) uses the random hash seed explained in the SO answer linked above. Every time we run the script, the ordering is randomized, which makes it impossible for us. We need a stable ordering for both the developer tests and so the two quarantined laptops will match each other. Hence OrderedDict.

In Python 3.7 the dict ordering is specified, and we wouldn't need OrderedDict anymore.